src/com/google/devtools/intellij/ijaas/OpenFileManager.java

```diff
@@ -103,8 +112,15 @@ private OpenedFile(String uri, int version, String text) {
       Pair<Editor, PsiFile> p =
           MoreWriteActions.computeAndWaitForDocument(
               () -> {
+                ;
+                String path = null;
```

Comment:
`= null` is not needed because the path is always initialized (or bail out by throwing an exception). Keep this uninitialized unless the default initialized value makes sense for all cases. Usage of uninitialized variables is easy to detect, vs initializing with a wrong value is not.